src/Sulu/Bundle/ContentBundle/Resources/public/js/components/single-internal-link/main.js

```diff
@@ -185,46 +184,27 @@ define([], function() {
         /**
          * initialize column navigation
          */
-        overlayOkCallback = function () {
```

Comment:
You are actually not allowed to remove that code. We've worked quite some time to make the overlay consistent, and that included that also a click on the ok button is necessary on this overlay.